.mci.yml

```diff
@@ -745,16 +748,17 @@ buildvariants:
           # signature.
           - name: compile_and_test_with_static_libs
 
-    - name: ubuntu1604-zseries
-      display_name: "s390x Ubuntu 16.04 (MongoDB Latest)"
+    - name: ubuntu1804-zseries
+      display_name: "s390x Ubuntu 18.04 (MongoDB Latest)"
       batchtime: 1440 # 1 day
       expansions:
           build_type: "Release"
+          cmake: "cmake"
```

Comment:
Looking for `cmake` in `/opt/cmake/bin/cmake` caused the test failures [here](https://evergreen.mongodb.com/task_log_raw/cxx_driver_ubuntu1804_zseries_compile_and_test_with_shared_libs_patch_3fe4528b7d33b5230f52895b3051be07cbfb0032_5f1aff467742ae3ba1b0e81c_20_07_24_15_34_56/0?type=T#L357) since that version depended on openssl 1.1.1. This defaults to the one in `/usr/bin`

Reply:
Acknowledged.

Reply:
Good find! Seems related to the issue I encountered attempting to upgrade the `power8-ubuntu1604` to Ubuntu 18.04: https://jira.mongodb.org/browse/CXX-2048. I'll likely use your new expansion to refer to the installation of cmake.